openpype/hosts/resolve/hooks/pre_resolve_setup.py

```diff
@@ -14,35 +13,91 @@ class ResolvePrelaunch(PreLaunchHook):
     app_groups = ["resolve"]
 
     def execute(self):
+        current_platform = platform.system().lower()
+
+        PROGRAMDATA = self.launch_context.env["PROGRAMDATA"]
```

Comment:
This will crash on linux or mac because they don't have `PROGRAMDATA`.

Reply:
good catch ;)